core/src/main/scala/com/pingcap/tispark/TiDBOptions.scala

```diff
@@ -106,6 +111,8 @@ object TiDBOptions {
   val TIDB_DATABASE: String = newOption("database")
   val TIDB_TABLE: String = newOption("table")
   val TIDB_REPLACE: String = newOption("replace")
-  val TIDB_SKIP_COMMIT_SECONDARY_KEY = newOption("skipCommitSecondaryKey")
-  val TIDB_SAMPLE_FRACTION = newOption("sampleFraction")
+  val TIDB_SKIP_COMMIT_SECONDARY_KEY: String = newOption("skipCommitSecondaryKey")
+  val TIDB_REGION_SPLIT_NUM: String = newOption("regionSplitNum")
+  val TIDB_ENABLE_REGION_SPLIT: String = newOption("enableRegionPreSplit")
+  val TIDB_REGION_SIZE: String = newOption("regionSize")
```

Comment:
I think this option should be set in spark static configuration. It is awkward to set a read-only TiKV configuration in tidbOption.

Reply:
better to get this value from tikv

Reply:
We cann't do that. The possible way to get region size is from pd but bringing new code may increase the complexity. I prefer the current way.

Reply:
agree with @birdstorm  better to move this parameter to spark config
if it's not so easy to implement, pls create an issue. 
`get from pd` is the right way. 
`get from config` is the easy way but work.

Reply:
Let's do it in another PR. This feature gets more and more complicated.